docs/app-dev/getting-started.md

```diff
@@ -31,17 +33,48 @@ echo export PATH=\"\$PATH:\$GOPATH/bin\" >> ~/.bash_profile
 
 Then run
 
-```sh
+```bash
 go get github.com/tendermint/tendermint
 cd $GOPATH/src/github.com/tendermint/tendermint
 make install_abci
 ```
 
-Now you should have the `abci-cli` installed; you'll notice the `kvstore`
-command, an example application written
-in Go. See below for an application written in JavaScript.
+Now you should have the `abci-cli` installed; run `abci-cli` to see the list of commands:
 
-Now, let's run some apps!
+```sh
```

Comment:
I checked the tool used for highlighting and bash and sh are aliases to the same grammar.

Reply:
My comment was rather that the snippet is a help (English) text, not a bash script, so by using "sh" or "bash", the rendered text is going to highlight words like "return", "echo", or "for" in the text, since they are reserved words in bash.